packages/react-sdk/src/components/StreamCall/ParticipantBox.tsx

```diff
@@ -192,6 +193,9 @@ export const ParticipantBox = (props: ParticipantBoxProps) => {
             {indicatorsVisible && !hasVideo && (
               <span className="str-video__participant_name--video-muted"></span>
             )}
+            {indicatorsVisible && isPinned && (
+              <span className="str-video__participant_name--pinned"></span>
+            )}
```

Comment:
I think it would be more intuitive if pinning can be done by clicking on this icon as well. Otherwise, pinning feels like a hidden feature of the ParticipantList.

Reply:
There is supposed to be a separate context menu living within `ParticipantBox` which will allow you to pin participants directly but that that's a question of a separate PR - I believe these little icons are for "status" only.

![image](https://user-images.githubusercontent.com/43254280/233390321-87db59eb-790a-43e7-a447-b61a5e96bacc.png)